indico/modules/events/abstracts/templates/management/abstract_list.html

```diff
@@ -73,15 +73,32 @@
                     </ul>
                 </div>
                 <div class="group">
-                    <a class="i-button icon-checkmark js-requires-selected-row js-dialog-action accept">
-                        {% trans %}Accept{% endtrans %}
+                    <a class="i-button icon-checkmark accept js-requires-selected-row"
+                       data-href="{{ url_for('.manage_judge_abstracts', event) }}"
+                       data-title="{% trans %}Accept selected abstracts{% endtrans %}"
+                       data-params-selector="#abstract-list input[name=abstract_id]:checked"
+                       data-params='{"action": "{{ actions.accept.name }}"}'
```

Comment:
Yes, if we change the enum member's name/value we'd have to change it here too. But it's so unlikely to happen that I don't think it's worth making code less readable for it. Especially considering how awful it is to use string operations to build JSON.

And anyway, how likely is it that someone would actually think about searching for `accept.name`? ;)